lib/administrate/field/associative.rb

```diff
@@ -11,6 +11,10 @@ def associated_class
         associated_class_name.constantize
       end
 
+      def short_plain_text
+        display_associated_resource
+      end
```

Comment:
You can probably just alias the methods.

Reply:
@BenMorganIO should I use `alias short_plain_text display_associated_resource` or `alias_method :short_plain_text, :display_associated_resource`?

Reply:
> This is because alias is a keyword and it is lexically scoped. It means it treats self as the value of self at the time the source code was read . In contrast alias_method treats self as the value determined at the run time.

- https://blog.bigbinary.com/2012/01/08/alias-vs-alias-method.html

They're both the same in this instance. The code is read at compile time so the performance should be the same. The lexical scoping presents no variation when doing it in the class. `alias` is a keyword and cannot be changed; just as much as you cannot change `do` or `end` in ruby. `alias_method` allows a person to change what happens since it is a method call.

Since the community has a preference for creating change in the system when possible, I would recommend `alias_method`. It's all down to philosophy here, and Ruby would opt for `alias_method`. If this was Elixir, we would be doing `alias`.

Reply:
@BenMorganIO 
As I understand now, If you alias some method, say, `AncestorField#short_plain_text` > `AncestorField#to_s` and override `to_s` in descendant (`DescendantField#to_s`) then `DescendantField#short_plain_text` will still point to `AncestorField#to_s` instead of `DescendantField#to_s` unless you redeclare the alias in `DescendantField`. I think it is not intuitive and I would stick with current implementation.

Reply:
Alright, these points are all very moot. I just felt refactoring to an alias would be nice, but if people need to extend that method, then this shouldn't be a problem.